src/wallet/wallet.ts

```diff
@@ -65,35 +92,47 @@ class HathorWalletServiceWallet extends EventEmitter {
     this.walletId = null;
     this.isSendingTx = false;
     this.txProposalId = null;
+    this.xpub = null;
 
     this.network = network;
+
+    this.authToken = null;
+
+    // TODO When we integrate the real time events from the wallet service
```

Comment:
Yes, we are. This PR was supposed to handle  only the token actions integration. I had to also do the authorization methods, otherwise it would not work. So I decided not to add any more features here and left the real time update for another PR.

Reply:
Ok. Just to confirm: Do we have issues to track both future developments?

Reply:
Yes